### PR TITLE
Fix the search form on home page

### PR DESCRIPTION
**Pull request by MarvinCrepin**

### CODE REVIEW

File: templates/space/index.html.twig [hunk 1]
```diff
@@ -1,57 +1,86 @@
...
-				<li class="spaceprice">
-					{{ random(50, 250) }}
-					€/jour
-				</li>
-			</ul>
-		</figcaption>
-	</figure>
-{% endfor %}{% endblock %}
+    <header class="header-home">
+        <div>
```
Comment (rocambille) [inline templates/space/index.html.twig]: the div looks useless here

File: templates/space/index.html.twig [hunk 2]
```diff
@@ -1,57 +1,86 @@
...
+            Un accélérateur de business,
+            <br>
+                créé par vous, pour vous.
+            </p>
+        </div>
+        {% if spaces %}
+            {% set space_count = 0 %}
+            {% for space in spaces %}
+                {% set space_count = space_count + 1 %}
+            {% endfor %}
```
Comment (rocambille) [inline templates/space/index.html.twig]: looks like a spaces|length

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
conflicts to resolve

**Comment by MarvinCrepin** [comment]
done

**Pull request merged**